src/components/boardComponent/boardComponent.js

```diff
@@ -2,29 +2,23 @@ import React, { Component } from 'react';
 import Square from '../squareComponent/squareComponent';
 import GameHeader from '../gameHeaderComponent/gameHeaderComponent';
 import './BoardComponent.css';
+import BoardView from '../../board/board';
 
 class Board extends Component {
```

Comment:
I think renaming this class to `BoardView` would remove some ambiguity, since you would no longer have two things named `Board`